### PR TITLE
Emit 'idle' on status request (closes #253)

### LINKED ISSUE #253: TestCafe terminates when idle page has not finished loading
Sometimes TestCafe terminates before  remote worker has finished loading of the idle page. It happens because we are emitting `idle` event just before idle page is rendered on the server. So we must wait until idle page is send to client.

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexanderMoskovkin @helen-dikareva

### CODE REVIEW

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
It seems we have an unstable test on ios 9.1. I'll look at this

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

File: src/browser-connection/index.js [hunk 1]
```diff
@@ -94,10 +95,6 @@ export default class BrowserConnection extends EventEmitter {
     }
 
     renderIdlePage () {
-        this.idle = true;
-
-        this.emit('idle');
-
         return Mustache.render(IDLE_PAGE_TEMPLATE, {
```
Comment (inikulin) [inline src/browser-connection/index.js]: Let's set `switchingToIdle = true` here instead of `getStatus`

**Comment by inikulin** [comment]
lgtm

**Comment by AndreyBelym** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**